rpc/block_svc_routines.c

```diff
@@ -3344,6 +3347,18 @@ block_create_1_svc_st(blockCreate *blk, struct svc_req *rqstp)
     goto out;
   }
 
+  if (GB_ASPRINTF(&glfs_alua,
+                  "%s/%s/alua create name=glfs_tg_pt_gp tag=1",
```

Comment:
Is tag equals to tg_pt_gp_id ? And is this always 1 for all the StorageObjects ?